contract_partner/README.rst

```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+================
+Contract Partner
+================
+
+This module was written to extend the functionality of contracts to support:
+* set contact and service partner in contract.
+* set contact and service partner from contract in invoice.
```

Comment:
Yes @dreispt I can change partner term by address, really I use partner for addresses.

"Service Partner" is the serviced address, it's usefull on inovices, for example in pest control company they do services on restaurant franchise, or simply one company with more than one  warehose, for this company is usefull to get address on invoice.

"Contact Partner" maybe usefull in other repo, it's used to set address to send printed invoice.